plugins/cache/redis.py

```diff
@@ -103,9 +104,9 @@ def __init__(self, *args, **kwargs):
             self._keys_set = self.get_option('_keyset_name')
             self._sentinel_service_name = self.get_option('_sentinel_service_name')
         except KeyError:
-            display.deprecated('Rather than importing CacheModules directly, '
-                               'use ansible.plugins.loader.cache_loader',
-                               version='2.0.0', collection_name='community.general')  # was Ansible 2.12
+            # TODO: remove once we no longer support Ansible 2.9
```

Comment:
And an issue here

Reply:
As discussed in IRC, it isn't a problem if this old code is left around even after Ansible 2.9 has gone EOL. These comments are left as reminders in case anyone is updating the plugin for other reasons.